tests/aws/services/kms/test_kms.snapshot.json

```diff
@@ -2238,5 +2238,37 @@
         }
       }
     }
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_re_encript[SYMMETRIC_DEFAULT-SYMMETRIC_DEFAULT]": {
+    "recorded-date": "19-05-2025, 07:54:39",
+    "recorded-content": {
+      "test_re_encript": {
+        "CiphertextBlob": "ciphertext-blob",
+        "DestinationEncryptionAlgorithm": "SYMMETRIC_DEFAULT",
+        "KeyId": "<key-id:1>",
+        "SourceEncryptionAlgorithm": "SYMMETRIC_DEFAULT",
+        "SourceKeyId": "<key-arn>",
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      }
+    }
+  },
+  "tests/aws/services/kms/test_kms.py::TestKMS::test_re_encript[RSA_2048-RSAES_OAEP_SHA_256]": {
+    "recorded-date": "19-05-2025, 07:54:41",
+    "recorded-content": {
+      "test_re_encript": {
+        "CiphertextBlob": "ciphertext-blob",
+        "DestinationEncryptionAlgorithm": "RSAES_OAEP_SHA_256",
+        "KeyId": "<key-id:1>",
+        "SourceEncryptionAlgorithm": "RSAES_OAEP_SHA_256",
+        "SourceKeyId": "<key-arn>",
+        "ResponseMetadata": {
+          "HTTPHeaders": {},
+          "HTTPStatusCode": 200
+        }
+      }
+    }
```

Comment:
This file is generated by the `snapshot` fixture, but it's not used in the test.

Could you have mistakenly removed it?